src/blahp/src/BUpdaterSGE.c

```diff
@@ -28,8 +28,8 @@ int main(int argc, char *argv[]){
     job_registry_entry *en;
     time_t now;
     time_t purge_time=0;
-    char *constraint[11];
-    char *constraint2[5];
+    char constraint[11];
```

Comment:
I'm a little concerned that the sizeof constraint is 11, but we use it like this:
`sprintf(constraint," %s",en->batch_id)`
where batch_id is
`char         batch_id[JOBID_MAX_LEN]`
and 
`#define JOBID_MAX_LEN▸  256`